src/remote.js

```diff
@@ -259,20 +276,20 @@
                       util.err(error);
                     }
                   )
-                },
-                function(error) {
-                  util.err(error);
                 }
-              )
+              },
+              function(err) {
```

Comment:
All right. Please confirm that the error message is not displayed like [object Object]